src/org/parosproxy/paros/extension/state/ExtensionState.java

```diff
@@ -52,6 +60,8 @@
 	private JCheckBoxMenuItem menuSessionTrackingEnable = null;
 
 	private ZapMenuItem menuResetSessionState = null;
+	private JCheckBoxMenuItem[] menuItemsMode = new JCheckBoxMenuItem[Mode.values().length];

```

Comment:
UI related components should not be initialised unless ZAP has GUI (there's the `initView` method for these kind of initialisations or in the `hook` method, if there's a view).